### PR TITLE
Migrate renovate bot configuration

### PR DESCRIPTION
**Pull request by hamza-56**
This PR addresses the warnings issued by the Renovate Bot configuration validator.

Renovate Bot has updated its default configuration preset from `config:base` to `config:recommended` starting from version 36.0.0 (released on August 23, 2023). 
**Reference:** https://github.com/renovatebot/renovate/pull/21136

Screenshot 2024-10-14 at 5 59 32 PM

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "extends": [
-    "config:base",
+    "config:recommended",
```
Comment (AfaqShuaib09) [inline renovate.json:3]: I'm not sure how this change will fix the issue.

I found an example of `renovate.json` where the config is set to "base," but the team is tagged correctly in the PR:
[Example](https://github.com/edx/edx-internal/blob/bf14ee530342747f066b466f82362f447ab93620/renovate.json#L5)

PR: [Renovate bot PR](https://github.com/edx/edx-internal/pull/11560)

Comment (hamza-56) [reply renovate.json:3]: @AfaqShuaib09 
This PR has been created to resolve the warnings generated by the Renovate config validator.

Comment (DawoudSheraz) [reply renovate.json:3]: We should double check if this config:base would work in new config before changing it to new value (or reference a relevant link that suggests using this new value).

Comment (hamza-56) [reply renovate.json:3]: Renovate Bot has updated its default configuration preset from `config:base` to `config:recommended` starting from version 36.0.0 (released on August 23, 2023). 
ref: https://github.com/renovatebot/renovate/pull/21136

**Review by DawoudSheraz — APPROVED** [review]

**Review by AfaqShuaib09 — APPROVED** [review]

**Pull request merged**